File: src/main/java/top/zbeboy/isy/web/bean/graduate/design/replan/DefenseGroupBean.java
package top.zbeboy.isy.web.bean.graduate.design.replan;

import lombok.Data;
import lombok.EqualsAndHashCode;
import top.zbeboy.isy.domain.tables.pojos.DefenseGroup;

import java.util.List;

/**
 * Created by zbeboy on 2017/7/11.
 */
@EqualsAndHashCode(callSuper = false)
@Data
public class DefenseGroupBean extends DefenseGroup {
    private Integer buildingId;
    private String buildingName;
    private String buildingCode;
    private String staffName;
    private String studentName;
    // 压缩组员信息
    private List<String> memberName;
}

File: src/main/java/top/zbeboy/isy/web/bean/graduate/design/replan/DefenseGroupBean.kt
package top.zbeboy.isy.web.bean.graduate.design.replan

import top.zbeboy.isy.domain.tables.pojos.DefenseGroup

/**
 * Created by zbeboy 2018-02-06 .
 **/
class DefenseGroupBean : DefenseGroup() {
    var buildingId: Int? = null
    var buildingName: String? = null
    var buildingCode: String? = null
    var staffName: String? = null
    var studentName: String? = null
    // 压缩组员信息
    var memberName: List<String>? = null
}